src/Driver/Mysqli/HostnameRequired.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Driver\Mysqli;
+
+final class HostnameRequired extends MysqliException
+{
+    public static function fromPersistentConnection() : self
```

Comment:
With that naming, I would expect a connection argument. How about this?
```suggestion
    public static function forPersistentConnection() : self
```

Reply:
Agree. Let's do that.